site/src/Librecores/ProjectRepoBundle/Resources/views/Default/project_search.html.twig

```diff
@@ -6,149 +6,132 @@ LibreCores Search
+  {# User Template #}
+  <script type="text/html" id="user-template">
+    {% verbatim %}
+          <h2><a href="/{{ username }}">{{{ _highlightResult.username.value }}}</a></h2>
+            <div>{{{ _highlightResult.name.value }}}</div>
+            <div><i>
+                joined on {{{ createdAt.date }}}
+              </i></div>
+            <hr/>
+    {% endverbatim %}
+  </script>
 
-    {% elseif search_query.type == 'orgs' %}
-      {% for org in orgs %}
-        <h2><a href="{{ path('librecores_project_repo_user_org_view', {'userOrOrganization':
-  org.getName()}) }}">{{ org.getDisplayName() }}</a></h2>
-        <div>{{ org.getName() }}</div>
-        <hr/>
-      {% else %}
-        {% if search_query.q is empty %}
-          <div>Search for organizations on LibreCores by entering a keyword above.</div>
-        {% else %}
-          <h2>Nothing found :-( Maybe try another search keyword?</h2>
-        {% endif %}
-      {% endfor %}
-    {% endif %}
-  </div>
-</div>
+  {# Organization Template #}
+  <script type="text/html" id="organization-template">
+    {% verbatim %}
+          <h2><a href="/{{ name }}">{{{ _highlightResult.displayName.value }}}</a></h2>
+            <div>{{{ _highlightResult.name.value }}}</div>
+            <hr/>
+    {% endverbatim %}
+  </script>
 
+  {# Project Template #}
+  <script type="text/html" id="projects-template">
+    {% verbatim %}
+        <h2><a href="/{{ parentUserName }}/{{ name }}">{{{ _highlightResult.parentUserName.value }}}/{{{ _highlightResult.name.value }}}</a></h2>
+            <div>{{{ _highlightResult.tagName.value }}}</div>
+            <div class="classification-container">
+            {{#_highlightResult.classifications}}
+               <a href="#" class="classifiation-tag">{{{ value }}}</a>
+            {{/_highlightResult.classifications}}
+            </div>
+            <div><i>
+              added on {{{ creationDetails }}}
+            </i></div>
+            <div class="librecores-project-list-metadata">
+              <span class="metric metric-lang">
+                <i class="fa fa-code"></i>
+                <span class="sr-only">Major language: </span>
+                {{{ _highlightResult.mostUsedLanguage.value }}}
+              </span>
+              <span class="updated">
+              <i class="fa fa-history"></i> Updated {{{ activityDetails }}}
+              </span>
+            </div>
+            <hr/>
+    {% endverbatim %}
+  </script>
+{% endblock %}
 
-{{ form_end(search_query_form) }}
-</div>
+{% block javascripts %}
+  {{ parent() }}
+  {% javascripts
+  'assets/js/instantsearch.js'
+  'assets/js/search.js'
+  filter="?jsqueeze" %}
+  <script src="{{ asset_url }}"></script>
+  {% endjavascripts %}
 {% endblock %}
 
 {% block pagejs %}
 {# do not use autofocus attribute to work around FF bug 712130 (FOUC) #}
+
 $(function() {
-  $("#{{ search_query_form.q.vars.id }}").focus();
+  searchFunctions();
+  options =  {
+  appId: 'ENTER_APPLICATION_ID',
```

Comment:
You can transclude the `site_algolia_app_id` here using twig globals

https://stackoverflow.com/a/8360126/2328165